build-setup/linux.md

```diff
@@ -239,58 +243,58 @@ to complete the build.
 
 ### CMake
 
-CMake version 3.19.2 is the minimum required to build ml-cpp. Download version 3.23.2 from <https://github.com/Kitware/CMake/releases/download/v3.23.2/cmake-3.23.2-Linux-x86_64.sh> and install:
+CMake version 3.30.5 is the minimum required to build ml-cpp. Download version 3.30.5 from <https://github.com/Kitware/CMake/releases/download/v3.30.5/cmake-3.30.5-linux-x86_64.sh> and install:
```

Comment:
Is CMake 3.30.5 the actual minimum version? Or just the one used here?

Reply:
It's not a minimum required indeed. It's just a decently recent version to support C++20 features.